consult.gemspec

```diff
@@ -24,7 +24,7 @@ Gem::Specification.new do |spec|
   spec.executables   = ['consult']
   spec.require_paths = ['lib']
 
-  spec.add_dependency 'diplomat', '~> 2.0.2'
+  spec.add_dependency 'diplomat', '~> 2.6.4'
```

Comment:
This is probably safe, and probably satisfactory for now, but I wonder if we'd benefit from being even less opinionated about the diplomat version. For example, if it was '~> 2.6`?